spec.html

```diff
@@ -27835,7 +27835,7 @@ <h1>String ( _value_ )</h1>
             1. If NewTarget is *undefined* and Type(_value_) is Symbol, return SymbolDescriptiveString(_value_).
             1. Let _s_ be ? ToString(_value_).
           1. If NewTarget is *undefined*, return _s_.
-          1. Return ? StringCreate(_s_, ? GetPrototypeFromConstructor(NewTarget, `"%StringPrototype%"`)).
+          1. Return StringCreate(_s_, ? GetPrototypeFromConstructor(NewTarget, `"%StringPrototype%"`)).
```

Comment:
Can `StringCreate` never return abruptly? If so, it should have an `!` - if not, it still needs a `?`

Reply:
> Can `StringCreate` never return abruptly?

Looks that way to me. If it can, there's something sneaky going on.

> If so, it should have an `!`

Well, that's a matter of taste. Personally, I don't think it'd be an improvement, but I'm not going to object if you put it in.

Reply:
The intent is to decorate every abstract operation with a ? or a !, if the completion record isn’t directly used.